libethcore/Transaction.cpp

```diff
@@ -130,6 +133,14 @@ Address const& TransactionBase::sender() const
 	return m_sender;
 }
 
+SignatureStruct const& TransactionBase::signature() const
+{ 
+	if (!m_vrs)
```

Comment:
does it make sense to throw here?
has  `Transaction` support now to be unsigned, it would maybe make sense to return null here